lib/tether/interfaces.go

```diff
@@ -33,7 +34,7 @@ type Operations interface {
 
 	SetHostname(hostname string, aliases ...string) error
 	Apply(endpoint *NetworkEndpoint) error
-	MountLabel(ctx context.Context, label, target string) error
+	MountLabel(ctx context.Context, label, target string, BlockDeviceMap map[string]fs.Fsinfo) error
```

Comment:
`BlockDeviceMap` as a map?  That's redundant or named incorrectly.  How about creating a struct `BlockDevices`, and hiding the map implementation inside it?